next/components/latest-articles.tsx

```diff
@@ -21,7 +27,22 @@ export function LatestArticles({ articles }: LatestArticlesProps) {
           </li>
         ))}
       </ul>
-      {!articles?.length && <p className="py-4">{t("no-content-found")}</p>}
+      <div className="flex items-center justify-center">
+        {!articles?.length && <p className="py-4">{t("no-content-found")}</p>}
+        {articles?.length && (
+          <LinkButtonWithStyles
+            href="/all-articles"
+            className="text-base mr-4 mt-4 inline-flex px-5 py-3"
+            variant="primary"
+          >
+            {t("all-articles")}
```

Comment:
adding a button to the existing "latest articles" to take the user to the all articles listing